a10_neutron_lbaas/v2/handler_pool.py

```diff
@@ -75,3 +75,14 @@ def delete(self, context, pool):
 
             handler_persist.PersistHandler(
                 c, context, pool, self._meta_name(pool)).delete()
+
+    def stats(self, context, pool):
+        result = {"stats": {}, "members": {}}
+        with a10.A10Context(self, context, pool) as c:
+            name = pool.id
+            if name is not None:
+                stats = c.client.slb.service_group.stats(name)
```

Comment:
I am trying to use this exact acos_client right now - however acos_client doesn't implement a stats method for service_group.

Only: get, create, update, all, delete, and delete_all under both 2.1 and 3.0  So as far as I can tell this will never work?  I also don't see a pull request adding this method.